src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int DEFAULT_RADIUS_CIRCLE = 10;
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int figureType = random.nextInt(5);
```

Comment:
Consider defining a constant for the number of figure types (5 in this case) to avoid using magic numbers. This will improve code readability and maintainability.